src/raftstore/store/fsm/peer.rs

```diff
@@ -518,6 +518,12 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
             SignificantMsg::Unreachable { to_peer_id, .. } => {
                 self.fsm.peer.raft_group.report_unreachable(to_peer_id);
             }
+            SignificantMsg::StoreUnreachable { store_id } => {
+                if let Some(peer_id) = util::find_peer(self.region(), store_id).map(|p| p.get_id())
+                {
+                    self.fsm.peer.raft_group.report_unreachable(peer_id);
```

Comment:
I have checked `report_unreachable` implementation. Seems here only do it for leaders is ok?